### ISSUE TITLE
[Screen Reader - PeoplePicker - NormalPeoplePicker]: NVDA Focus is not moving to the input field of NormalPeoplePicker, when navigated in NVDA browse mode.

### ISSUE THREAD

**Issue by dikshantbansal98** [microsoft/fluentui#15060]
**Component:** PeoplePicker

**PWD Impact:**
Visually challenged users who rely on screen readers are missing information as NVDA Focus is not moving to the NormalPeoplePicker input field, when navigated in NVDA browse mode.

**Test Environment:**
OS : Win 10 2004 (OS Build 19041.450)
Browser:  New Edge 85.0.564.44
URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker
Tools used: NVDA 2020.2

**Repro Steps:**
Step 1: Enable NVDA and Open the above URL in New Edge browser.
Step 2: Now navigate through the page in NVDA browse mode and verify whether the focus is moving to input field of NormalPeoplePicker or not.

**Actual Result:**
NVDA Focus is not moving to the input field, when navigated in NVDA browse mode.
Note : Same issue is observed for the compact and controlled people picker component on the page as well

**Expected Result:**
NVDA Focus must move to input field, when navigated in NVDA browse mode.

**MAS References:** 
MAS 2.1.1 - Keyboard
WCAG Reference Link: https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by paulgildea** [comment]
@dikshantbansal98 Thanks for the feedback.

**Comment by kolaps33** [comment]
I took look on the issue and quickly investigated. 

Looks like NVDA is broken when there is some element in accessibility tree between the "combobox" and "input" element. 
So I just put the role=presentation on the remaining  <div> which was visible in accessibility tree. 

Then NVDA starts to work for me, pl. see screenshot:
![combobox_fix](https://user-images.githubusercontent.com/22620150/94259265-7bded100-ff2e-11ea-817a-1f2562ea8737.png)

**Comment by jenniaynlynn** [comment]
I support a team that found a similar issue using Talkback and Bluetooth keyboard on Android10.  You can enter 1 person, but can't add any other individuals.  Focus only goes to the previously selected person and doesn't even let you focus on the input at all.  We found it does work with swipe when Talkback is on...just not with Bluetooth keyboard.

**Comment by smhigley** [comment]
Closed via #17202
